src/main/docs/guide/configuration/availableOptions.adoc

```diff
@@ -21,6 +21,7 @@ PACKAGE +
 PROTECTED +
 PUBLIC | Default: `PUBLIC`
 |`*micronaut.openapi.json.format*` | Is this property true, output file format will be JSON, otherwise YAML. | Default: `false`
+|`*micronaut.openapi.generator.extensions.enabled*` | If this property is 'true', then generated OpenAPI specification will be with extensions for OpenAPI Generator and the generated client according to this specification will be much more accurate than without it. For example, enumerations will be described with extensions `x-enum-varnames`, `x-enum-descriptions` and `x-deprecated` | Default: `false`
```

Comment:
can we link to these extensions ?

Reply:
Here described only two of them: https://openapi-generator.tech/docs/templating/ 

`x-enum-varnames`, `x-enum-descriptions`

`x-deprecated` - only in PR: https://github.com/OpenAPITools/openapi-generator/pull/19673

I think you need to understand that the open api generator is a half-dead project. They don't merge PRs for months (or even years). You shouldn't expect any standard from it, there hasn't been any active development in it for over a year.

I look at what extensions are supported in the spring generator and try to add new ones so that our generator can generate perfect code.

Reply:
My issue, is how are people going to discover how to use these non standard extensions?

Reply:
Rightly. I'll add documentation tomorrow

Reply:
@sdelamo documentation added